wondrous-app/components/Dashboard/boards/index.tsx

```diff
@@ -1,13 +1,13 @@
 import { useLazyQuery, useQuery } from '@apollo/client';
-import { ViewType } from 'types/common';
-import { bindSectionToColumns, sectionOpeningReducer } from 'utils/board';
-import _ from 'lodash';
-import { useRouter } from 'next/router';
-import React, { useCallback, useEffect, useReducer, useState } from 'react';
+import { useMe } from 'components/Auth/withAuth';
+import Boards from 'components/Common/Boards';
+import BoardsActivity from 'components/Common/BoardsActivity';
+import { FilterItem, FilterItemIcon, FilterItemName } from 'components/Common/Filter/styles';
+import CreateDaoIcon from 'components/Icons/createDao';
+import CreatePodIcon from 'components/Icons/createPod';
 import {
   GET_JOIN_ORG_REQUESTS,
   GET_JOIN_POD_REQUESTS,
-  GET_PER_STATUS_TASK_COUNT_FOR_USER_BOARD,
```

Comment:
afaik this query fetches the counts for tasks in user board, do we still update that if we don't use this?

Reply:
@Lamperoyge Let me know if this answers your question or not.

No, the task counts on the panel won't be updated if we don't use `GET_PER_STATUS_TASK_COUNT_FOR_USER_BOARD`.

<img width="600" alt="image" src="https://user-images.githubusercontent.com/8164667/179867608-e2d887b4-b292-480c-9b46-16a75be7c65d.png">